### PR TITLE
Feat/home userflow

### PR DESCRIPTION
**Pull request by nottolivc**
# Description
Updated homepage to match UX recommendations and added a loading component
Fixes # (issue)
Homepage
## Type of change
Clean up for homepage
Please delete options that are not relevant.

- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)

## Change Status

- [ x ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ x] Locally 
- [ x] Deployed

# Checklist

- [ x ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ x ] My code has been reviewed by at least one peer
- [ x ] I have commented my code, particularly in hard-to-understand areas
- [ x ] I have made corresponding changes to the documentation
- [ x ] My changes generate no new warnings
- [  x] I have added tests that prove my fix is effective or that my feature works
- [ x ] New and existing unit tests pass locally with my changes
- [ x ] There are no merge conflicts

### CODE REVIEW

**Review by BPitts8019 — COMMENTED** [review]
The Context should be created outside of the App component. We want to have a single reducer with dispatchers to update the context. In the context folder there is a place to create our context and reducers, then we can put the context provider in the AppProviders component.

File: src/App.tsx [hunk 1]
```diff
@@ -7,15 +7,59 @@ import Logout from "./views/_shared/Logout";
...
+//       case "LOGOUT":
+//          localStorage.clear();
+//          return {
+//             ...state,
+//             isAuthenticated: false,
+//          };
+//       default:
+//          return state;
+//    }
+// };
```
Comment (BPitts8019) [inline src/App.tsx:37]: This should be done in the context folder

File: src/App.tsx [hunk 2]
```diff
@@ -7,15 +7,59 @@ import Logout from "./views/_shared/Logout";
...
+
+function App() {
+   // const [state, dispatch] = React.useReducer(reducer, initialState);
+   return (
+      <>
+         {/* <AuthContext.Provider{{...state:any}}>
```
Comment (BPitts8019) [inline src/App.tsx:43]: We should have all Providers in the AppProviders component

File: src/views/_shared/Footer.tsx [hunk 3]
```diff
@@ -6,8 +6,8 @@ const Footer = (props: any) => {
          <nav>
             <a href="https://partnerd.dev">Home</a>
             <a href="https://partnerd.dev">Contact</a>
-            <a href="https://partnerd.dev">Login</a>
-            <a href="https://partnerd.dev">Register</a>
+            <a href="https://partnerd.dev/login">Login</a>
+            <a href="https://partnerd.dev/register">Register</a>
          </nav>
```
Comment (BPitts8019) [inline src/views/_shared/Footer.tsx:11]: These should be relative links rather than hardcoded to the production site

File: src/App.tsx (hunk 2 above)
Comment (nottolivc) [reply src/App.tsx:43]: agreed I will remove that

File: src/views/_shared/Footer.tsx (hunk 3 above)
Comment (BPitts8019) [inline src/views/_shared/Footer.tsx:11]: These should be relative URLs rather than hard coded to production URLs

File: src/App.tsx (hunk 1 above)
Comment (nottolivc) [reply src/App.tsx:37]: Agreed I will do a quick merge then remove them and merge App with no commentary

**Pull request merged**